### ISSUE TITLE
Missing component: Badge

### ISSUE THREAD

**Issue by jyng** [urbit/indigo-react#68]
I would like to use the Badge UI Element in indigo-react, but has yet to exist. 

<img width="407" alt="Screen Shot 2020-09-24 at 3 02 52 PM" src="https://user-images.githubusercontent.com/9557907/94204985-2c899980-fe77-11ea-9a1f-2b3b160fd445.png">

What I'm trying to implement in my app

<img width="328" alt="Screen Shot 2020-09-24 at 3 03 04 PM" src="https://user-images.githubusercontent.com/9557907/94205023-3a3f1f00-fe77-11ea-8c90-c44c507b6875.png">

**Comment by jyng** [comment]
Here's the code I'm using for Badge.tsx

```
import * as React from "react";
import { Text } from "@tlon/indigo-react";

const Badge = props =>
<Text backgroundColor="scales.black50" color="white" p="1" borderRadius="2" {...props}/>

export default Badge;
```

**Comment by g-a-v-i-n** [comment]
Resolved in 1.2.8
